docs/ROADMAP.md

```diff
@@ -0,0 +1,47 @@
+# Roadmap Interchain Security
+
+_Lastest update: June 20th, 2022_
+
+This document endeavors to inform the wider Cosmos community about plans and priorities for the work on Interchain Security (ICS). This roadmap should be read as a high-level guide, rather than a commitment to schedules and deliverables. The degree of specificity is inversely proportional to the timeline. We will update this document periodically to reflect the status and plans.
+ 
+## Q2 - 2022
+
+- Finalize core ICS features that enable a provider chain to provide security to multiple consumer chains.
+  - https://github.com/cosmos/interchain-security/issues/35
+  - https://github.com/cosmos/interchain-security/issues/27
+  - https://github.com/cosmos/interchain-security/issues/39
+- Create a minimum viable consumer chain (MVCC) that enables testing core ICS features.
+  - https://github.com/cosmos/interchain-security/issues/139
+- Create in-memory test drive based on the ibc-go testing framework.
+  - https://github.com/cosmos/interchain-security/pull/126
+- Finalize ICS spec
+  - https://github.com/cosmos/ibc/issues/670
+
+## Q3 - 2022
+
+- Differential testing of ICS that tests the core ICS features (excluding distribution, channel initialization, consumer chain removal, expired clients).
+  - https://github.com/cosmos/interchain-security/issues/137
+- Create a governance-enabled consumer chain that enables a native token to be staked for governance. 
+  - https://github.com/cosmos/interchain-security/issues/141
+  - https://github.com/cosmos/interchain-security/issues/82
+- Create a CosmWasm-enabled consumer chain.
+  - https://github.com/cosmos/interchain-security/issues/143
+- Enable validators to set different consensus keys for different consumer chains.
+  - https://github.com/cosmos/interchain-security/issues/26
+- Enable the chains to be easily restarted in case they halt.
+  - https://github.com/cosmos/interchain-security/issues/158
+  - https://github.com/cosmos/interchain-security/issues/156
+  - https://github.com/cosmos/interchain-security/issues/125
+  - https://github.com/cosmos/interchain-security/issues/121
+- Fix remaining issues
```

Comment:
I would move anything relating to ICS on the Hub up to the top of the priority list

Reply:
Not that this list is necessarily in order of priority, but move it up still

Reply:
It looks like everything past this point is ICS on the provider, so I would basically just flip the two halves of the list